### PR TITLE
unify configuration and commands

### PR DESCRIPTION
**Pull request by wkloucek**
## Description
This PR:
- adds missing commands for some services (which previously couldn't be started except with `ocis server`)
- use tags on the config structs to get config values from the environment (instead of `mappings.go`)
- unify common config parts like `Service.Version` 
- ensures, that each config file will only be parsed once
- changes the behaviour of subcommands: eg. `ocis accounts` will no longer run a server, but display the help. In order to run the server one needs to run `ocis accounts server`.
- the version subcommand now displays the version / compile time and the version of running services (context dependent which services are displayed)
- moved a lot of stuff around in its own packages (eg. config.Parse(...))
- moved a lot of stuff around in its own files (eg. default config, http config, grpc config, ...)

Out of scope:
- `./storage/` was not touched, this will be done in a separate PR

## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- tested configuration loading behavior locally according to https://owncloud.dev/ocis/config/

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by ownclouders** [comment]
:boom: Acceptance test [<strong>localApiTests-apiSpaces-ocis</strong>](https://drone.owncloud.com/owncloud/ocis/8419/20/1) failed. Further test are cancelled...

**Comment by butonic** [comment]
hm I may have added the machine auth command in https://github.com/owncloud/ocis/pull/2910

do you need a review after fixing the merge conflict?

File: accounts/pkg/command/health.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+			if err != nil {
+				logger.Fatal().
+					Err(err).
+					Msg("Failed to request health check")
+			}
+
+			defer resp.Body.Close()
+
+			if resp.StatusCode != 200 {
```
Comment (C0rby) [inline accounts/pkg/command/health.go:39]:
```suggestion
			if resp.StatusCode != http.StatusOK {
```

**Review by C0rby — APPROVED** [review]
Just one small things. :+1:

File: accounts/pkg/command/health.go (hunk 1 above)
Comment (wkloucek) [reply accounts/pkg/command/health.go]: :+1:

**Pull request merged**